hs/_asm/undocumented/spaces/internal.so.dSYM/Contents/Info.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what do you need internal.so stuff for/how did it get in here? We obviously cannot add dynamic libraries directly to this repo, ignoring that it's arch specific, it's also a security nightmare :)